src/Directory.Build.props

```diff
@@ -45,7 +45,7 @@
 
   <PropertyGroup>
     <!-- Used by SIL.ReleaseTasks below -->
-    <ChangelogFile>$(SolutionDir)/CHANGELOG.md</ChangelogFile>
```

Comment:
Note: This fixed an error caused when trying to pack MAUI, because we call `dotnet pack` on just the one csproj, so there's no solution dir.